docs/advanced-features/workflows.md

```diff
@@ -354,6 +354,78 @@ With this Schema lists such as
 `["Fixed 1", "Any string", 123]` 
 are valid but `["Any string", 123]` are invalid.
 
+### Objects
+Quilt also supports the [`object` data type](https://json-schema.org/understanding-json-schema/reference/object.html).
```

Comment:
This section is confusing. Eliminate or rework opening. It's not about Quilt but about the validators we support. I'd nest these under `### Example properties` and say "The following example show how you can specify complex properties such as objects and arrays with arbitrary nesting." And then `####` h4 for subsequent sections.

Reply:
Fixed